components/react/hooks/src/useMountedState/index.js

```diff
@@ -1,6 +1,6 @@
 import {useCallback, useEffect, useRef} from 'react'
 
-export default function useMountedState(value, delay) {
+export default function useMountedState() {
```

Comment:
I have doubts about this hook, why not returning this one as a boolean directly more than a getter function?

Reply:
It is using the reference for the behavior and no-useEffect should depend on the state of the hooks inner value, so we only should return a memoized callback targetting to the value (no other component observers will be refreshed when its inner status is updated)

take a look to the [https://github.com/streamich/react-use/blob/master/src/useMountedState.ts](https://github.com/streamich/react-use/blob/master/src/useMountedState.ts) code example

Reply:
Sure i understand, what i mean is that even returning directly the boolean value and not a getter it shouldn't notify and rerender any component observing this value